pkg/config/config.go

```diff
@@ -102,9 +103,10 @@ func InstallFlags(nodeIP string) []string {
 		"--no-taints",
 		"-c", runtimeconfig.PathToK0sConfig(),
 	}
+	flags = append(flags, ProfileInstallFlag(cfg))
 	flags = append(flags, AdditionalInstallFlags(nodeIP)...)
 	flags = append(flags, AdditionalInstallFlagsController()...)
-	return flags
+	return flags, nil
```

Comment:
if you only ever return nil better to leave it without an error in the signature